xesmf/util.py

```diff
@@ -212,6 +212,31 @@ def split_polygons_and_holes(polys):
     return exteriors, holes, i_ext, i_hol
 
 
+def densify_polys(polys, max_length=1.0):
```

Comment:
On my side, this works:
``` python3
import shapely as shp
p = shp.box(0, 0, 10, 10)
p2 = shp.box(0, 0, 12, 12)
shp.segmentize([p, p2], 1)
```
and returns an array of segmentized polygons. Indeed, we may be able to get rid of `densify_polygons`!

Reply:
I've added a note in the nb about the shapely segmentize function, so ok to remove the function.